pyne/endf.pyx

```diff
@@ -40,11 +40,10 @@ libraries = {0: 'ENDF/B', 1: 'ENDF/A', 2: 'JEFF', 3: 'EFF',
              4: 'ENDF/B High Energy', 5: 'CENDL', 6: 'JENDL',
              31: 'INDL/V', 32: 'INDL/A', 33: 'FENDL', 34: 'IRDF',
              35: 'BROND', 36: 'INGDB-90', 37: 'FENDL/A', 41: 'BROND'}
-FILE1_R = re.compile(r'1451 *\d{1,5}$')
-CONTENTS_R = re.compile(' +\d{1,2} +\d{1,3} +\d{1,10} +')
-SPACE66_R = re.compile(' {66}')
-NUMERICAL_DATA_R = re.compile('[\d\-+. ]{80}\n$')
+FILE1_R = re.compile(r'1451[ \d]{4}\d$')
```

Comment:
This still isn't quite right.

The old one will match incorrectly on  `1451  5` - not enough spaces and/or digits.
This one will match incorrectly on `1451  5 5` - a mixture of the correct number of spaces or digits.
Perhaps it should be:
```suggestion
FILE1_R = re.compile(r'1451(?=[ ]*\d+$)[ \d]{5}$')
```
This adds a look head to ensure that there is one group of 0+ spaces and the one group of 1+ digits before the new line, and then makes sure there are 5 of them, combined.

Reply:
It might be useful to add tests for these regex expressions that don't rely on a file, but simply document a set of developer-constructed cases that either should or should not match and then a test function that shows if/when they do/don't.

Reply:
Let me sleep on it...